roles/sahara/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+dependencies:
+  - role: openstack-database
+    database_name: sahara
+    package_alternatives: "{{ sahara.package.console_scripts }}"
```

Comment:
This variable isn't defined, you should protect that with a |default, or define it to be an empty list.